packages/ui/vue/create-headless-vue/template/README.md

```diff
@@ -7,10 +7,10 @@ This template should help get you started developing your Coveo Search Page with
 If you generated this project using the [Coveo CLI](https://github.com/coveo/cli), your project should come with a `.env` file with the necessary credentials for you to get started right away.
 If you didn't use the Coveo CLI, copy `.example.env` to `.env` and fill it out with the proper information:
 
-- `VITE_APP_PLATFORM_URL`: the URL you usually use to use the Coveo APIs, usually `https://platform.cloud.coveo.com`
-- `VITE_APP_ORGANIZATION_ID`: Your Coveo Organization ID. Check the doc to see [how to retrieve your organization ID](https://docs.coveo.com/en/148/manage-an-organization/retrieve-the-organization-id)
-- `VITE_APP_API_KEY`: Your API Key for this Org. We recommend the 'Search' API Key preset.
-- `VITE_APP_USER_EMAIL`: Your email address. Used for security identities and Analytics.
+- `VITE_COVEO_PLATFORM_URL`: the URL you usually use to use the Coveo APIs, usually `https://platform.cloud.coveo.com`
+- `VITE_COVEO_ORGANIZATION_ID`: Your Coveo Organization ID. Check the doc to see [how to retrieve your organization ID](https://docs.coveo.com/en/148/manage-an-organization/retrieve-the-organization-id)
+- `PRIVATE_APP_API_KEY`: Your API Key for this Org. We recommend the 'Search' API Key preset.
```

Comment:
A bit late but did you mean `SERVER_COVEO_API_KEY`

Reply:
Ah yeah good catch

Reply:
#1089